src/abstractions/serialization/SerializationWriterProxyFactory.cs

```diff
@@ -15,7 +15,11 @@ public class SerializationWriterProxyFactory : ISerializationWriterFactory
         /// The valid content type for the <see cref="SerializationWriterProxyFactory"/>
         /// </summary>
         public string ValidContentType { get { return _concrete.ValidContentType; } }
-        private readonly ISerializationWriterFactory _concrete;
+
+        /// <summary>
+        /// The concrete factory to wrap.
+        /// </summary>
+        protected readonly ISerializationWriterFactory _concrete;
```

Comment:
This should follow the naming conventions: `Concrete` and while at it, we might want to give it a better name now that it's exposed. Like "RealSerializationWriterFactory"

Reply:
`protected readonly ISerializationWriterFactory ProxiedSerializationWriterFactory;`